bedrock/firefox/templates/firefox/firstrun/membership-b.html

```diff
@@ -0,0 +1,81 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/firstrun/base.html" %}
+
+{% block body_id %}firstrun-membership{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_title %}{{ _('Welcome to Firefox') }}{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'firefox_firstrun_membership' %}
+{% endblock %}
+
+{% block js %}
+  {% javascript 'firefox_firstrun_membership' %}
+{% endblock %}
```

Comment:
Is this placed here intentionally to be blocking?

Reply:
This isn't blocking per-say. The place where it appears in the source order is defined in the base template (at the bottom before closing body tag). Saying that, I'll move it to the bottom of this template too, just to avoid confusion.

Reply:
Ah ok. I was wondering whether the ordering mattered in Jinja or whether you can "implement" these blocks in any order and they will still be stuck into the normal source location.

I do agree though that it would look less confusing if done at the end of the file. 👍